src/components/FormikAutocomplete.jsx

```diff
@@ -5,16 +5,28 @@ import PropTypes from 'prop-types';
 
 /* eslint-disable react/jsx-props-no-spreading */
 function FormikAutocomplete(props) {
-  const { label, id, labelStyle, formik, placeholder, ...otherProps } = props;
+  const { label, id, labelStyle, formik, placeholder, options, ...otherProps } =
+    props;
   const values = getIn(formik.values, id, []);
+
+  const formatTags = (vals) => {
+    const newVals = new Set(
+      vals.map((v) => v.toLowerCase().replace(/\s+/g, '-'))
+    );
+    formik.setFieldValue(id, Array.from(newVals));
+  };
+
   return (
     <>
       <InputLabel sx={labelStyle}>{label}</InputLabel>
       <Autocomplete
         id={id}
         multiple
         value={values}
-        onChange={(e, val) => formik.setFieldValue(id, val)}
+        options={options}
```

Comment:
This doesn't seem to be set by anything. What's the difference between this and `values`?

Reply:
`values` are the tags inputted into the text area. 
Since it's an `autocomplete`, it requires `options`. 

I could remove this and pass an empty array from the `scholarshipForm`.